apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -150,6 +153,11 @@ class UnitOverviewTopRow extends React.Component {
     let completedProfessionalLearningCourse =
       isProfessionalLearningCourse && unitProgress === COMPLETED;
 
+    let hasPrintingOptionsDropwdown =
```

Comment:
Super nit: I wonder if we want to reframe this comment from `hasPrinting...` to `displayPrinting...` or `allowPrinting...`. With `has`, it reads like the PrintingOptionsDropdown is something that already exists with this row, vs. `display` or `allow` reads like this is the boolean which is determining whether the dropdown is exists.
(Re-reading that, I'm not sure if I was success in communicating my thoughts, so let me know if you want to chat about this one!)

Reply:
I like either of those better.  I noticed `has` as a common naming convention, but you're right that it doesn't quite communicate what we'd like.